sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -1049,3 +1106,111 @@ case class GpuToCpuCollectBufferTransition(
     projection.apply(InternalRow.apply(arrayData)).getBytes
   }
 }
+
+/**
+ * Base class for overriding standard deviation and variance aggregations.
+ * This class is a GPU-based equivalent of `CentralMomentAgg` aggregation class in Spark.
```

Comment:
nit: it is not 100% the same because it is only doing the M2 portion of it. It is missing m3 and m4, which is not needed in the aggregations that we support.